cmd/main.go

```diff
@@ -23,6 +23,8 @@ import (
 	"net/http/pprof"
 	"os"
 
+	"github.com/thanos-community/thanos-operator/internal/pkg/controllers_metrics"
```

Comment:
nit: but we could keep the ordering here and elsewhere and group the controller packages together :)